components/fieldgroup/stories/template.js

```diff
@@ -53,7 +55,22 @@ export const Template = (
 					[`${rootClass}InputLayout`]: true,
 				})}
 			>
-				${renderContent(items, { args: { isReadOnly, isRequired }, context })}
+				${inputType === "radio" ?
+					items.map((item) =>
+					Radio({
+					...item,
+					isReadOnly,
+					isRequired,
+					name: "field-group-example",
+					customClasses: [`${rootClass}-item`],
+					}, context))
+					: items.map((item) =>
+					CheckBox({
+					...item,
+					isReadOnly,
+					isRequired,
+					customClasses: [`${rootClass}-item`],
+				}, context))}
```

Comment:
This is a total nitpick, and doesn't really block this PR at all, but could we fix up some of the indentation? 

```
items.map((item) =>
	Radio({
		...item,
		isReadOnly,
		isRequired,
		name: "field-group-example",
		customClasses: [`${rootClass}-item`],
	}, context))
```